server/src/main/java/org/apache/druid/metadata/IndexerSQLMetadataStorageCoordinator.java

```diff
@@ -1833,26 +1900,26 @@ private Set<DataSegment> announceHistoricalSegmentBatch(
         for (DataSegment segment : partition) {
           final String now = DateTimes.nowUtc().toString();
           preparedBatch.add()
-              .bind("id", segment.getId().toString())
-              .bind("dataSource", segment.getDataSource())
-              .bind("created_date", now)
-              .bind("start", segment.getInterval().getStart().toString())
-              .bind("end", segment.getInterval().getEnd().toString())
-              .bind("partitioned", (segment.getShardSpec() instanceof NoneShardSpec) ? false : true)
-              .bind("version", segment.getVersion())
-              .bind("used", usedSegments.contains(segment))
-              .bind("payload", jsonMapper.writeValueAsBytes(segment))
-              .bind("used_status_last_updated", now);
+                       .bind("id", segment.getId().toString())
```

Comment:
Since this is a sensitive PR for a new feature, let's revert all these formatting changes for now. Better to do reformatting in refactor PRs.

Please revert all formatting changes in other lines in this class too.

Reply:
Ah sorry, I must have auto-indented the whole file